bin/spryker-sdk.sh

```diff
@@ -119,6 +119,10 @@ case $MODE in
     if [[ -z "$SPRYKER_SDK_ENV" || $SPRYKER_SDK_ENV == 'prod' ]]; then
         docker-compose -f "${SDK_DIR}/docker-compose.yml" run --rm spryker-sdk "$ARGUMENTS"
     else
+        if [[ ! -f "${SDK_DIR}/docker-compose.dev.yml" ]]; then
+           echo "SPRYKER_SDK_ENV=$SPRYKER_SDK_ENV environment is not available for installer. Remove SPRYKER_SDK_ENV env variable if it exists or set it value to \"prod\" by running \"export SPRYKER_SDK_ENV=prod\""
```

Comment:
Can we please use ` for code fencing. " is not proper for this.

Reply:
This was not fixed now in the way it should be.

Reply:
I'm not sure, but afaik ` can be recognized as a command. Example
```shell
[spryker@SDK_CLI]$ x="`pwd`"
[spryker@SDK_CLI]$ echo $x
/data
[spryker@
```

Reply:
done